e2e/morse_relay_test.go

```diff
@@ -0,0 +1,105 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	// morseConfigFile is the name of the config file under "e2e" directory, that contains
+	// the config for a PATH instance that uses Shannon as the relaying protocol.
```

Comment:
NIT: should it say `uses Shannon as the relaying protocol.`?

Reply:
Thank you. Fixed.